pptx/enum/dml.py

```diff
@@ -312,6 +312,7 @@ class MSO_THEME_COLOR_INDEX(XmlEnumeration):
         XmlMappedEnumMember("LIGHT_2", 4, "lt2", "Specifies the Light 2 theme color."),
         XmlMappedEnumMember("TEXT_1", 13, "tx1", "Specifies the Text 1 theme color."),
         XmlMappedEnumMember("TEXT_2", 15, "tx2", "Specifies the Text 2 theme color."),
+        XmlMappedEnumMember("PLACEHOLDER", 16, "phClr", "Specifies to use the Style Color."),
```

Comment:
Took me a while to figure this out as well.   In a shape, you can apply a style.  If for example, you are applying a Fill Style, it will provide an `idx` and a color.  This means that powerpoint should use the fill style from the theme, using the index defined by the `idx`.  Within that style that is defined in the theme, it will define a color from the `MSO_THEME_COLOR_INDEX` enum.  If that is the `PLACEHOLDER` value, then the color used is the one defined back from the shape, when it calls the Style.  It's basically a variable color.